src/components/NavigationBar/NavigationBar.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+	import type { Menu } from '$models/menu';
+
+	import Banner from './Banner.svelte';
+	import MenuPane from './MenuPane.svelte';
+	import MenuList from './MenuList.svelte';
+	import SideMenuButton from './SideMenuButton.svelte';
+	import SideMenuPane from './SideMenuPane.svelte';
+	import SideMenuList from './SideMenuList.svelte';
+
+	import PoliticianIcon from '$components/icons/PoliticianIcon.svelte';
+	import LawIcon from '$components/icons/LawIcon.svelte';
+	import VoteIcon from '$components/icons/VoteIcon.svelte';
+	import WeVisIcon from '$components/icons/WeVisIcon.svelte';
+	import NavigationPane from './NavigationPane.svelte';
+	import SearchInput from './SearchInput.svelte';
+
+	//import logoLong from '/static/images/logo/pw-long-white.png';
+	//import logoShort from '/static/images/logo/pw-short-white.png';
+
+	export let searchActionLink = '/search';
```

Comment:
Please remove this prop in this component and  `SeachInput` since the search function won't be implemented this way (Just remove the action from the form for now). It's gonna be an autocomplete which will be implemented in another issue.